landscape/categories.md

```diff
@@ -0,0 +1,56 @@
+## Proposed Security Landscape Structure
+In this section of the document, we outline our proposal for the structure of the Security Landscape.
+
+The bullets in **bold** are the primary categories in the Security Landscape and the bullets in _italics_ are the
+sub-categories. The remaining text is a description of each sub-category.
+
+- **App definition and development**
+  - _Static Code Analysis_
+    - Inspecting code for OWASP vulnerabilities, pen testing, etc
+  - _Dependency analysis_
+    - Checking OS for vulnerabilities (updates available? Reports from https://www.us-cert.gov/ etc), image scanning
+    - Check for vulnerabilities in dependent libraries
+    - Check for maintenance of dependent projects => increased potential for vulnerabilities
+  - _Pipeline security testing_
+    - Tools that facilitate automated security testing in pipelines; eg for functional security tests of authn and authz,
+      tests of known potential weaknesses or misconfigurations
+
+- **Identity & Access Control**
+  - _Identity_
+    - SPIFFE, identity providers, OpenID, LDAP, Okta
+  - _Access Controls_
+    - Controls within the orchestration layer to provide minimum required access to orchestrator resources.
+    - Authentication / Authorization
+      - OAuth, IAM
+    - Privilege & Access Management
+      - Managing privileges in the platform for humans and machines, access controls, audit logging, privilege revocation,
+        etc
+    - Network Policy & Protection
+      - Firewalls, network segmentation, network policy
+  - _Service access_
+    - Service admittance & admission controllers
+      - Allowing new service instances to join existing services, and preventing insecure application instances from running.
+    - Storage access
+  - _Key and Certificate Management_
+    - Provisioning of access to required services for application instances, service accounts, orchestrator access to shared components, etc
+    - Provisioning of SSL certs for application instances
+
+- **Privacy**
```

Comment:
Where does data sovereignty fit into in this category? For example, keeping European data siloed or preventing data with different levels of classification from mixing (at rest or in motion)

Reply:
@dshaw and I thought data sovereignty was same as data location, so made that change